inference-engine/thirdparty/clDNN/src/impls/onednn/pooling_onednn.cpp

```diff
@@ -15,10 +15,103 @@
 namespace cldnn {
 namespace onednn {
 
+struct pooling_onednn : typed_primitive_onednn_impl<pooling, dnnl::pooling_forward::desc> {
+    using parent = typed_primitive_onednn_impl<pooling, dnnl::pooling_forward::desc>;
+    using parent::parent;
+
+protected:
+    std::unique_ptr<primitive_impl> clone() const override {
+        return make_unique<pooling_onednn>(*this);
+    }
+
+    static std::shared_ptr<dnnl::pooling_forward::desc> get_pooling_descriptor(const pooling_node& arg) {
+        auto prim = arg.get_primitive();
+
+        auto& input = arg.get_dependency(0);
+        auto spatials_rank = cldnn::format::spatial_num(input.get_output_layout().format);
+
+        auto stride = onednn::convert_spatials(prim->stride, spatials_rank);
+        auto kernel = onednn::convert_spatials(prim->size, spatials_rank);
+        auto pad_l = onednn::convert_spatials(prim->input_offset, spatials_rank);
+        auto pad_r = onednn::convert_spatials(prim->pad_end, spatials_rank);
```

Comment:
Is it just overwritten below? Then it is not necessary? I think ngraph must have done the calculation already. In that case, we may confirm that new calculation is equal to previous calculation.